app/src/screens/bank.js

```diff
@@ -6,12 +6,49 @@ import { Container, Section } from '../ui/layout';
 import { Text } from '../ui/typography';
 import NavBar from '../ui/nav-bar';
 import Card from '../ui/card';
+import Toast from '../ui/toaster/toast';
+
+function GameHistory() {
+  let { notice, history } = useGame();
+
+  if (!notice || !history.length) { return null; }
+
+  return (
+    <Section data-test-history>
+      <Text data-test-history-heading>History</Text>
+
+      <div>
+        <Section compact>
+          <Toast
+            type={'message'}
+            message={notice.message}
+          />
+        </Section>
+        {history.map(({ notice }, index) => {
+          if(!notice) { return; }
+
+          return (
```

Comment:
Let's reduce returns here and just do `return notice && (`

As long as the notice is never `0`, which it should never be anyway, it'll short circuit to the same outcome.